src/fr.rs

```diff
@@ -62,7 +63,7 @@ impl From<Fr> for BlsScalar {
         // so convert any jubjub scalar to a BLS' Scalar should always be
         // safe.
         assert!(
-            bool::from(bls_scalar.is_some()),
+            bool::from(bls_scalar.is_ok()),
```

Comment:
`is_ok` returns already a boolean, this is an unnecessary conversion, just have `assert!(bls_scalar.is_ok(), "Failed to convert a Scalar from JubJub to BLS");`